src/sas/sascalc/dataloader/readers/cansas_reader.py

```diff
@@ -1169,11 +1163,13 @@ def _write_notes(self, datainfo, entry_node):
         :param entry_node: lxml node ElementTree object to be appended to
 
         """
-        if len(datainfo.notes) == 0:
-            node = self.create_element("SASnote")
-            self.append(node, entry_node)
-        else:
-            for item in datainfo.notes:
+        from sas.sasview import __version__ as version
+        node = self.create_element("SASnote")
+        self.write_text(
+            node, 'Data file generated by SasView v{0}'.format(version))
```

Comment:
Thinking toward splitting sascalc into its own repo, if another program is using sascalc to write canSAS data it is not correct to say that the data is generated by SasView. You will need a different method to indicate how the data was generated, such as adding it to datainfo in the GUI or adding `notes=None` to the `write` method.

If you stick with the current scheme, any reason not to put the import at the top of the file?

Reply:
I'll look into changing the write method for this PR, but if that doesn't happen, I will add that to the larger loader PR.